src/pages/home/report/ReportActionCompose.js

```diff
@@ -134,12 +134,14 @@ class ReportActionCompose extends React.Component {
         this.getInputPlaceholder = this.getInputPlaceholder.bind(this);
         this.getIOUOptions = this.getIOUOptions.bind(this);
         this.addAttachment = this.addAttachment.bind(this);
-
         this.comment = props.comment;
-        this.shouldFocusInputOnScreenFocus = canFocusInputOnScreenFocus();
+
+        // React Native will retain focus on an input for native devices but web/mWeb behave differently so we have some focus management
+        // code that will refocus the compose input after a user closes a modal or some other actions, see usage of ReportActionComposeFocusManager
+        this.shouldOverrideRnFocusBehavior = shouldOverrideRnFocusBehavior();
```

Comment:
No need create an instance variable IMO just call the function wherever

Reply:
Hm my only reason for keeping it an instance variable is that we don't have to duplicate the comment in each place we call it, not a huge deal though and I don't have a very strong opinion on it.

Reply:
ok we could also export a static variable from a module and leave the same commen where it gets imported. No reason to make it a function and set its return value to an instance variable.

Reply:
ah yeah not a bad idea